raiden/exceptions.py

```diff
@@ -159,3 +159,15 @@ class TransportError(RaidenError):
 
 class ReplacementTransactionUnderpriced(RaidenError):
     """Raised when a replacement transaction is rejected by the blockchain"""
+
+
+class ChannelOutdatedError(RaidenError):
+    """
+    Raised when an action is invoked on a channel whose
+    identifier has been replaced with a new channel identifier
+    due to a close of current channel and a re-open of a new one.
+    """
+    def __init__(self, error_msg, current_channel_id, new_channel_id):
```

Comment:
Some time ago we decided to not pass args to constructors. Just pass the message parameterized as a string at the point of exception initialization. If you check you will other exceptions do the same.